test.py

```diff
@@ -26,6 +27,7 @@
     parser = argparse.ArgumentParser(description="Seastar test runner")
     parser.add_argument('--fast',  action="store_true", help="Run only fast tests")
     parser.add_argument('--name',  action="store", help="Run only test whose name contains given string")
+    parser.add_argument('--skip',  action="append", help="Skip tests whose name contains given string")
```

Comment:
how about just exposing a flag allowing user to pass random parameters for `ctest`? like
```
./test.py -- --test-timeout 3600
```

Reply:
good point. done.
also i've dropped the dns test patch in favor of PR #2683